<launch>
  <param name="use_sim_time" value="false"/>
  <arg name="model" />
  <arg name="sim_model" default="-m DRC_NO_HANDS" />
  <param name="/atlas/robot_description" textfile="$(find ihmc_msgs)/description/drc_no_hands.urdf" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" >
    <remap from="joint_states" to="/atlas/outputs/joint_states" />
    <remap from="robot_description" to="/atlas/robot_description" />
  </node>
  <node name="IHMCAtlasAPI" pkg="ihmc_msgs" type="IHMCAtlasAPI" required="true" output="screen" cwd="node">
  </node>
</launch>
